tools/uploader-function/README.md

```diff
@@ -0,0 +1,25 @@
+# Uploader Function
+
+This solution uses the Azure Function framework to allow uploads to a DICOM Service.
+
+Steps:
+## Create app service plan
+```
+az functionapp plan create --resource-group {resource-group-name} --name {app-service-plan-name} --location {REGION} --number-of-workers 1 --sku EP1 --is-linux
+```
+
+## Create function app
+```
```

Comment:
Here call out:
- Same `{resource-group}` as above
- Same `{app-service-plan-name}` as above
- `{storage-account-name}` is the storage account that the function app will use during the import, NOT the source storage account
- Its safe to ignore warnings about lack of credentials accessing the container registry (it's publicly accessible)